latex/cardano-ledger.tex

```diff
@@ -36,20 +36,65 @@
 \includeAgda{Ledger/Chain}
 
 \section{Properties}
-\includeAgda{Ledger/Utxo/Properties}
+This section presents the properties of the ledger that we have formally proved in
+Agda or plan to do so in the near future.  We indicate in which Agda module each
+property is formally stated and (possibly) proved. A ``Claim'' is a property that is
+not yet proved, while a ``Theorem'' is one for which we have a formal proof.
+
+\subsection{Preservation of Value}
+\inputAgda{Ledger/Ledger/Properties/PoV}
+\inputAgda{Ledger/Utxo/Properties/PoV}
+\inputAgda{Ledger/Certs/Properties/PoV}
+\inputAgda{Ledger/Certs/Properties/PoVLemmas}
+
+\subsection{Invariance Properties}
+\Cref{thm:ChainGovDepsMatch} and \cref{lem:LedgerGovDepsMatch,lem:EpochGovDepsMatch} assert that
+a certain relation is an invariant of the \CHAIN{}, \LEDGER{}, and \EPOCH{} rules, respectively.
```

Comment:
```suggestion
a certain predicate is an invariant of the \CHAIN{}, \LEDGER{}, and \EPOCH{} rules, respectively.
```
Relation or predicate on states? Here is called relation but a bit below predicate.